ln-simln-jamming/src/analysis.rs

```diff
@@ -57,17 +57,60 @@ impl Serialize for Record {
                 .forwarding_outcome(self.forward.amount_in_msat, self.forward.incoming_endorsed),
         )?;
         state.serialize_field(
-            "revenue_threshold",
-            &self.decision.reputation_check.revenue_threshold,
+            "incoming_revenue_threshold",
```

Comment:
WDYT about naming these by the reputation check we're doing rather than the directions?

```
rep_outgoing_threshold = outgoing_reputation.reputation
rep_outgoing_reputation = outgoing_reputation.threshold
rep_outgoing_risk = outgoing_reputation.htlc_risk

rep_incoming_threshold = incoming_reputation.reputation
rep_incoming_reputation = incoming_reputation.threshold
rep_incoming_risk = incoming_reputation.htlc_risk
```

I think that this is how you'd look at the data (checking one direction of reputation check at a time), and IMO it makes it a little easier to group these values.